packages/parse5-html-rewriting-stream/package.json

```diff
@@ -6,6 +6,9 @@
     "author": "Ivan Nikulin <ifaaan@gmail.com> (https://github.com/inikulin)",
     "contributors": "https://github.com/inikulin/parse5/graphs/contributors",
     "homepage": "https://github.com/inikulin/parse5",
+    "scripts": {
+        "build": "rm -r dist; tsc"
```

Comment:
for sake of improving DX we may want to use rimraf or something so windows based contributors can get this building

Reply:
Thanks for flagging! I realised I could use `tic --build --clean` instead.